src/buildercore/terraform.py

```diff
@@ -550,14 +558,17 @@ def destroy(stackname, context):
     terraform_directory = join(TERRAFORM_DIR, stackname)
     shutil.rmtree(terraform_directory)
 
+# TODO: not a great function name. 'stack_tform_path' ? 'tform_stackfile_path' ?
 def _file_path(stackname, name, extension='tf.json'):
```

Comment:
Terraform has no concept of stacks, but each of these is a `configuration file`, so `_configuration_file_path`.

With hungarian notation it could be `_tf_configuration_file_path` would be clear for some cases, but it is really responsible for any file that ends up there, even if it is a `.vcl` (Varnish Configuration Language). `tform` is not in use anywhere.

Reply:
(`_single_leading_underscore` is from [PEP 8](https://www.python.org/dev/peps/pep-0008/))